client/my-sites/upgrades/checkout-thank-you/index.jsx

```diff
@@ -126,16 +126,9 @@ var CheckoutThankYou = React.createClass( {
 					{ this.productRelatedMessages() }
 				</Card>
 
-				<Card className="checkout-thank-you__get-support">
-					<div className="checkout-thank-you__get-support-text">
-						<h3 className="checkout-thank-you__support-heading">
-							{ this.translate( 'Questions? Need Help?' ) }
-						</h3>
-						<p className="checkout-thank-you__support-related-messages">
-							{ this.supportRelatedMessages() }
-						</p>
-					</div>
-				</Card>
+				<CheckoutThankYouFooter
+					isDataLoaded={ this.isDataLoaded() }
+					receipt={this.props.receipt} />
```

Comment:
I noticed that as well. Looks like my change wasn't taken into account. I'll fix that.